### DISCUSSION TITLE
What's the difference between atoti and activeUI?

### DISCUSSION THREAD

**Discussion by GameTn** [atoti/atoti#548] [Q&A] [answered]
Hi , i saw that activeUI is very similar to atoti especially in the visualizations , what's the difference between them ?

**Comment by bloussou** [comment, marked as answer]
atoti is packaging a version of ActiveUI to use it as user interface. You cannot load data in ActiveUI since it is only a UI. You can read a clear description in the introduction of the [ActiveUI documentation](https://activeviam.com/activeui/documentation/5.0.9/docs/getting-started).
